utils/capabilities.c

```diff
@@ -104,10 +104,15 @@ gbSetCapabilties(void)
     goto out;
   }
 
+  if (globalCapabilities) {
+    GB_FREE(globalCapabilities);
+  }
+
```

Comment:
I won't object, but IMO this is not needed, as we only user of global var 'globalCapabilities' is in gbSetCapabilties() and this won't be only called once.

But nothing wrong in satisfying coverity requirement :-)

Reply:
Done